model/publishing/delivery/tasks/DeployTestEnvironments.php

```diff
@@ -84,6 +86,9 @@ protected function compileTest(\core_kernel_classes_Resource $env, \core_kernel_
                 'instances' => $test->getUri(),
             ], \tao_helpers_File::createTempDir());
             $packagePath = $exportReport->getData();
+            if(is_array($packagePath) && isset($packagePath['path'])){
```

Comment:
I think it would be better to fix root cause of this issue and return consistent data from exported.
To fix that we can remove this line - https://github.com/oat-sa/extension-tao-testqti/blob/master/models/classes/export/class.TestExport.php#L123 and fix place where it's used (and related tests) - https://github.com/oat-sa/extension-tao-operations/blob/master/model/ExportTestMetaData/TestDataExporter.php#L76